buildSrc/src/main/kotlin/Dependencies.kt

```diff
@@ -1,6 +1,7 @@
 object Dependencies {
   val apacheCommonsLang3 = "org.apache.commons:commons-lang3:3.11"
   val apacheCommonsPool2 = "org.apache.commons:commons-pool2:2.11.1"
+  val arrowCore = "io.arrow-kt:arrow-core:1.2.1"
```

Comment:
```suggestion
```

I don't think this library is necessary for this change and warrants broader discussion before introducing to Misk.

Reply:
will remove.